flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -881,24 +883,21 @@ class GroupWindowedTable(
     * }}}
     */
   def select(fields: Expression*): Table = {
```

Comment:
Hi @fhueske , i think it will be a more complex situation when window involves. However, i want to keep this change simple and straightforward, and open another issue to track the scenario when involving windows. Does this make sense to you?